components/src/Label/Label.js

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import PropTypes from "prop-types";
+import theme from "../theme";
+import Text from "../Type/Text";
+import Box from "../Box/Box";
+
+export const LabelText = props => {
+  return (
+    <Text
+      display="inline"
+      mb={ 0 }
+      fontSize={ theme.fontSizes[1] }
+      { ...props }
+    />
+  );
+};
+
+LabelText.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]),
+};
+
+LabelText.defaultProps = {
+  children: null,
+};
+
+export const RequirementText = props => {
+  return (
+    <Text
+      display="inline"
+      mb={ 0 }
+      ml={ 2 }
+      fontSize="12px"
+      color="darkGrey"
+      { ...props }
+    />
+  );
+};
+
+RequirementText.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]),
+};
+
+RequirementText.defaultProps = {
+  children: null,
+};
+
+export const HelpText = props => {
+  return (
+    <Text
+      mb={ 0 }
+      fontSize={ theme.fontSizes[0] }
+      lineHeight={ theme.lineHeights.smallTextBase }
+      { ...props }
+    />
+  );
+};
+
+HelpText.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]),
+};
+
+HelpText.defaultProps = {
+  children: null,
+};
+
+export const FormatText = props => {
+  return (
+    <Text
+      mb={ 0 }
+      fontSize="12px"
+      lineHeight="16px"
+      color="darkGrey"
+      { ...props }
+    />
+  );
+};
+
+FormatText.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]),
+};
+
+FormatText.defaultProps = {
+  children: null,
+};
+
+const Label = props => {
+  const {
+    labelText,
+    requirementText,
+    helpText,
+    formatText,
+  } = props;
```

Comment:
The last idea is consistent with our current rules, although I think we should introduce a new rule that "all additional props not aimed at specific sub-components should be passed along to the parent wrapper component". This way it is an expectation that margins, color, etc applied to the molecular component will apply to the component in general rather than a specific sub-component. Let me know what you think

Reply:
@clemensp Also I agree deconstructing at the parameter level is much cleaner!